.circleci/config.yml

```diff
@@ -365,6 +365,8 @@ jobs:
       - attach_workspace: { at: "." }
       - check-changed:
           patterns: contracts-bedrock,op-node
+      - setup_remote_docker:
```

Comment:
This doesn't seem to be enough to set up docker for the kontrol job

Reply:
Yea trying to figure out why. It seems to be sufficient for the `kontrol-tests` job but not the `contracts-bedrock-tests` job, but the "Setup a remote Docker engine" step is still not present for the bedrock tests . Maybe related to `- attach_workspace: { at: "." }`, but still investigating...

<img width="1516" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/17163988/8691b04b-f5c1-4629-a28f-770e118b3ad5">

Reply:
I had `setup_remote_docker` under the `contracts-bedrock-tests` job, not the `contracts-bedrock-checks` job 🤦

Pushed 4d4135fda which should fix it

Reply:
yep, docker is running. Next error is an invalid symlink in typescript land

Reply:
Investigating—I can reproduce this locally by running the `make-deployment-summary.sh` and `run-kontrol.sh` scripts in container mode

Reply:
Ok got it all passing with a few changes, so resolving this thread